applications/models.py

```diff
@@ -83,6 +84,44 @@ class Organisation(models.Model):
     )
 
 
+class ApplicationRound(models.Model):
+    name = models.CharField(
+        verbose_name=_("Name"),
+        max_length=255,
+    )
+    reservation_units = models.ManyToManyField(
+        ReservationUnit,
+        verbose_name=_("Reservation units"),
+        related_name="application_rounds",
+    )
+
+    application_period_begin = models.DateField(
+        verbose_name=_("Application period begin"),
+    )
+    application_period_end = models.DateField(
+        verbose_name=_("Application period end"),
+    )
+
+    reservation_period_begin = models.DateField(
+        verbose_name=_("Reservation period begin"),
+    )
+    reservation_period_end = models.DateField(
+        verbose_name=_("Reservation period end"),
+    )
+
+    purposes = models.ManyToManyField(
```

Comment:
It has been mentoined in discussions that there will be certain purposes for the application rounds so I thought that this could be used for displaying them and also maybe filtering reservation units with these purposes when staff is creating the application round.

I thought that it would be better practice to choose purposes for the application period and then choose reservation units for it rather than first choosing from all reservation units which may have many purposes and then determine which of those purposes to use for this application round.

Reply:
Yeah, that makes sense.